src/audiovideocontroller/AudioVideoControllerState.ts

```diff
@@ -106,6 +106,9 @@ export default class AudioVideoControllerState {
   // itself correspond to transceivers; 0 in this list corresponds to an inactive tranceiver.
   videoSubscriptions: number[] | null = null;
 
+  // Total number of videos streams that can be subscribed to.
+  videoSubscriptionLimit: number = 0;
```

Comment:
Nit: Comment where this number comes from (because it will always and only come from the backend), that it may change, and where it is used (particularly since `AudioVideoControllerState` is already sort of a global store, and easily abusible; if possible, we at least want to warn future abusers of this value about how it is set).